### ISSUE TITLE
logos are all the same

### ISSUE THREAD

**Issue by cchristelis** [kartoza/feti#339]
# Problem

Currently all logos are the same. (I'm not sure if it is the TVET icon)

# Proposed solutions

- Scrape an logo. 
- or use a more generic placeholder logo.

**Comment by cchristelis** [comment]
@marchturnbull, which default icon would you consider most appropriate?

**Comment by cchristelis** [comment]
Scraping could be done in the following way: if a web address is present and the underlying site is working? Then we look for the logo in the top left corner. A case where this could work would be:

http://ncap.careerhelp.org.za/learningprovider/1d0549a7-4fe1-4923-80ec-63348e437f44/learningpathway/08b0659b-c050-4a6c-8cce-637232da50fb/occupation/eb463869-fece-4952-98fb-0b71d2907257/generalqualification/a5b7c66a-006a-44cb-8e57-7de771f1fcbc
http://www.dut.ac.za

**Comment by meomancer** [comment]
This also part of this : https://github.com/kartoza/feti/issues/271

**Comment by meomancer** [comment]
- For showing icon, updating model to save campus icon, here is the PR : https://github.com/kartoza/feti/pull/349 (merged)
- For scraping, here is the PR : https://github.com/kartoza/feti/pull/353

@cchristelis @marchturnbull, still need default icon

**Comment by marchturnbull** [comment]
I am not sure of the question.  The most important differences in the icons should be between Public and Private - do we still have a database entry that allows us to identify that?

**Comment by cchristelis** [comment]
Hi @marchturnbull, I mistyped here and called the logos icons :p. I have changed the title and problem statement accordingly. 

Currently we are using the same placeholder logo: 
<img width="569" alt="screen shot 2016-12-08 at 1 41 32 pm" src="https://cloud.githubusercontent.com/assets/2319504/21008907/8bd1c82e-bd4c-11e6-8ec4-682f07cea65b.png">

This logo is not a very good placeholder, since it is actually the logo of an institute. So the question was, in the case we can't scrape a logo, is there a FETI, Provincial etc logo that we should use?
